pdh_check/README.md

```diff
@@ -26,11 +26,7 @@ Run the [Agent's status subcommand][5] and look for `pdh_check` under the Checks
 
 ### Metrics
 
-See [metadata.csv][6] for a list of metrics provided by this integration.
```

Comment:
get rid of link then?

Reply:
This is already being deleted from the doc in this section. However, we cannot delete it permanently from the repo as it's required as a placeholder to avoid breaking scripts downstream in dogweb.

Reply:
No.. I mean link id `6` is no longer used

Reply:
Fixed, thanks for that catch 😸